src/models/resultsSerializer.ts

```diff
@@ -256,16 +280,26 @@ export default class ResultsSerializer {
     /**
      * Open the saved file in a new vscode editor pane
      */
-    public openSavedFile(filePath: string): void {
+    public openSavedFile(filePath: string, format: string): void {
         const self = this;
-        let uri = vscode.Uri.file(filePath);
-        self._vscodeWrapper.openTextDocument(uri).then((doc: vscode.TextDocument) => {
-            // Show open document and set focus
-            self._vscodeWrapper.showTextDocument(doc, 1, false).then(undefined, (error: any) => {
+        if (format === 'excel') {
+            // This will not open in VSCode as it's treated as binary. Use the native file opener instead
+            // Note: must use filePath here, URI does not open correctly
+            opener(filePath, undefined, (error, stdout, stderr) => {
```

Comment:
I didn't have a chance to test - will test on Linux but likely post checkin. This simply calls the default file open action on the machine, so it should prompt for a sane default. I handle any error callbacks from it so it should report errors if something does go wrong.